hangman/game.py

```diff
@@ -1,25 +1,87 @@
 from .exceptions import *
+import random
 
 # Complete with your own, just for fun :)
 LIST_OF_WORDS = []
 
-
-def _get_random_word(list_of_words):
-    pass
-
+""" Mask word"""
 
 def _mask_word(word):
-    pass
+  if word =='':
+    raise InvalidWordException
+    
+  else: 
+    mask_word = '*' * len(word) 
+    return mask_word
 
+""" Uncover word """
 
 def _uncover_word(answer_word, masked_word, character):
-    pass
-
+  answer_word = answer_word.lower()
+  character = character.lower()
+  list_ans = list(answer_word)
+  list_mask = list(masked_word)  
+  
+  if ((answer_word or masked_word) == '') or (len(masked_word)!= len(answer_word)):
+    raise InvalidWordException
+  
+  elif len(character) > 1:
+    raise InvalidGuessedLetterException
+    
+  elif character in answer_word:
+      i = 0
+      for i in range(len(list_ans)):
```

Comment:
as mentioned earlier on `string` being a collection like `list`, you can use `for` loop like this:-
`for letter in answer_word:`
`if letter is character:`